modules/kubernetes-addons/aws-pca-issuer/locals.tf

```diff
@@ -0,0 +1,47 @@
+locals {
+  name                 = "aws-pca-issuer"
+  service_account_name = "${local.name}-sa"
+
+  default_helm_config = {
+    name        = local.name
+    chart       = local.name
+    repository  = "https://cert-manager.github.io/aws-privateca-issuer"
+    version     = "0.1.2"
+    namespace   = local.name
+    description = "AWS PCA Issuer helm Chart deployment configuration."
+    values      = local.default_helm_values
+    timeout     = "1200"
+  }
+
+  default_helm_values = []
+
+  helm_config = merge(
+    local.default_helm_config,
+    var.helm_config
+  )
+
+  set_values = [
+    {
+      name  = "serviceAccount.create"
+      value = "false"
+    },
+    {
+      name  = "serviceAccount.name"
+      value = local.service_account_name
+    }
+  ]
+
+  irsa_config = {
+    create_kubernetes_namespace       = true
+    kubernetes_namespace              = local.name
+    create_kubernetes_service_account = true
+    kubernetes_service_account        = local.service_account_name
+    irsa_iam_policies                 = [aws_iam_policy.aws_pca_issuer.arn]
+    irsa_iam_permissions_boundary     = var.irsa_iam_permissions_boundary
```

Comment:
This is going to move to `addon_context`. Please keep an eye on this [PR](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/pull/351).